var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -46,11 +48,13 @@ class PyCelery(PythonPackage):
     # 'cosmosdbsql',
     # 's3',
 
+    depends_on('python@3.7:', type=('build', 'run'), when="@5.2.3:")
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
 
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-redis@3.2.0:', when='+redis', type=('build', 'run'))
+    depends_on('py-hiredis@0.1.3:', when="+hiredis", type=('build', 'run'))
     depends_on('py-sqlalchemy', when='+sqlalchemy', type=('build', 'run'))
 
     depends_on('py-click@7.0:7', when="@5.0.0:5", type=('build', 'run'))
```

Comment:
Many of these dep versions have changed and need to be updated, see the files in `requirements/*`